README.md

```diff
@@ -1,6 +1,10 @@
 # SEM Size Analysis
 
-_A tool for segmenting SEM images for the purposes of analyzing size distributions of nanoparticles
+SAEMI is a tool for obtaining a Size Analysis of particles in Electron Microscopy Images. It uses an image segmentation model 
+trained through a neural network to detect (nano)particle sizes and locations. From there, it then obtains a histogram of the 
```

Comment:
Be specific (not necessarily here, but somewhere) about which neural net you used (a u-net), and which library (fastai).